### PR TITLE
Correct offsets for the iPhone 5,3 9.3.2

### PR DESCRIPTION
**Pull request by EGYbkgo9449**
Correct offsets for the iPhone 5,3 9.3.2

### CODE REVIEW

File: Trident/offsetfinder.c [hunk 1]
```diff
@@ -16,1104 +16,1161 @@
...
-    determineTarget("iPhone4,1", "9.3.4", iPhone41_iOS934);
-    determineTarget("iPhone5,2", "9.2", iPhone52_iOS920);
-    determineTarget("iPhone5,2", "9.2.1", iPhone52_iOS921);
-    determineTarget("iPhone5,2", "9.3.2", iPhone52_iOS932);
-    determineTarget("iPhone5,3", "9.2.1", iPhone53_iOS921);
-    determineTarget("iPhone5,3", "9.3.2", iPhone53_iOS932);
-    determineTarget("iPhone5,3", "9.3.3", iPhone53_iOS933);
-    determineTarget("iPhone5,4", "9.3.2", iPhone54_iOS932);
-    determineTarget("iPad2,1", "9.1", iPad21_iOS910);
+	// iPhone 4S (iPhone4,1)
```
Comment (benjamin-42) [inline Trident/offsetfinder.c]: @EGYbkgo9449 could you please only commit specific changes for iPhone5,3 iOS 9.3.2? And also check the indentation, should be 4 spaces

Comment (EGYbkgo9449) [reply Trident/offsetfinder.c]: Done, sry for that

File: Trident/offsetfinder.c [hunk 2]
```diff
@@ -1,4 +1,10 @@
-//
+uint32_t find_OSSymbol_getMetaClass(void) {
```
Comment (benjamin-42) [inline Trident/offsetfinder.c]: Where are other offsets? Please just add a case for your device + OS combination

File: Trident/offsetfinder.c [hunk 3]
```diff
@@ -1117,3 +1123,14 @@ uint32_t find_mac_proc_check(void) {
         default: abort();
     }
 }
+
+    switch (target_environment) {
```
Comment (benjamin-42) [inline Trident/offsetfinder.c]: Should be part of the main switch for this function

Comment (EGYbkgo9449) [reply Trident/offsetfinder.c]: I've wrote ur offsetfinder.c and changed only the wrong offsets -_-

**Comment by benjamin-42** [comment]
I rebased your work to get only one commit and to remove the changes made at the top of the file (you removed the comment), changes are available in master.

**Pull request closed without merging**